spinbot/gh/client.py

```diff
@@ -40,8 +40,14 @@ def get_label(self, repo, name, create=True):
 
         return label
 
+    def repos(self):
```

Comment:
`@property`?

Reply:
I've avoided that for now... will do a pass to introduce where necessary